test-junit/jvm/src/main/scala/zio/test/junit/ZTestJUnitRunner.scala

```diff
@@ -28,6 +28,7 @@ class ZTestJUnitRunner(klass: Class[_]) extends Runner with Filterable with Boot
 
   private lazy val spec: AbstractRunnableSpec = {
     klass
+      .getDeclaredConstructor()
```

Comment:
Not sure why this was not failing the CI builds because it tests fail to compile on my machine locally w/o this change.